src/schema/rules/checks/nifti.yaml

```diff
@@ -49,3 +49,18 @@ XformCodes0:
     - nifti_header != null
   checks:
     - nifti_header.qform_code != 0 || nifti_header.sform_code != 0
+
+NiftiPEDir:
+  issue:
+    code: NIFTI_PE_DIRECTION_CONSISTENCY
+    message: |
+      Phase encoding direction is inconsistent with `dir-` entity.
```

Comment:
```suggestion
      Phase encoding direction may be inconsistent with `dir-` entity.
```

Reply:
I'm not opposed to this, but I don't understand the case where this is triggered and the direction isn't inconsistent. We can expand on this text to explain why it's inconsistent in general terms, though we don't have a way to vary the text based on the values of fields.

Reply:
```suggestion
      The PhaseEncodingDirection and affine orientation indicate
      that the entity value (one of AP, PA, RL, LR, SI, IS) may be inaccurate.
```